### PR TITLE
Bug 1825007: Fix tuned reload behaviour on SIGHUP for invalid tuned profiles.

### PR DESCRIPTION
**Pull request by jmencak**
This PR fixes the behaviour when reloading tuned daemon on SIGHUP.  Tuned pods
will now correctly display error messages from tuned pods when invalid tuned
profile is detected.  More importantly, upon correcting the custom tuned
profile the new profile itself will be reloaded.

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/openshift/openshift-tuned
 COPY . .
 RUN make build
 
-FROM registry.svc.ci.openshift.org/ocp/4.0:base
+FROM registry.svc.ci.openshift.org/ocp/4.3:base
```
Comment (russellb) [inline Dockerfile.rhel7:6]: nit: This doesn't appear related to the bug fixed and described in the commit message.  It'd be better to split this into another commit.

It's obvious enough though, so you don't need to redo it this time ...

Comment (russellb) [reply Dockerfile.rhel7:6]: (context: i'm reviewing backport PRs as patch manager this week)

Comment (jmencak) [reply Dockerfile.rhel7:6]: Understood and thanks, Russell.  It would be great if this could merge this week (customer case).  Let me know if I need to change anything this time round.  https://github.com/openshift/cluster-node-tuning-operator/pull/128 is associated with the case and needs to go to 4.3 too.
